### PR TITLE
Extend Apps docuentation (#131)

### PR DESCRIPTION
**Pull request by korycins**
* Add App commands section. Extend mermaid styles

* Update styling and docs

* Use shorter syntax, don't suggest using `saleor` in App ids

* Add documentation for graphql App section

* Update object names

* Add section to explain differences between install and create

* Rename celery worker to Backgorun worker

* Rename activate flag to shorter

* rename homepageUrl to homepageOnMarketUrl

* Create section for local apps and third-party apps

* Revert "rename homepageUrl to homepageOnMarketUrl"

This reverts commit 481154da7d114ecde9289f0ed6dbd7ab5a97bf2a.

* Apply changes after review

Co-authored-by: Patryk Zawadzki <patrys@room-303.com>

### CODE REVIEW

File: docs/developer/extending/apps.mdx [hunk 1]
```diff
@@ -102,26 +424,142 @@ mutation {
...
+      field
+      message
+      code
+    }
+  }
+}
+```
+
+
+The query returns simillar response:
```
Comment (maarcingebala) [inline docs/developer/extending/apps.mdx:546]:
```suggestion
The query returns a similar response:
```

File: docs/developer/extending/apps.mdx [hunk 2]
```diff
@@ -102,26 +424,142 @@ mutation {
...
+      },
+      "appErrors": []
+    }
+  }
+}
+```
+### Deactivating an App
+
+To deactivate an app, use the `appDeactivate` mutation. This mutation is only available to staff users with the `MANAGE_APPS` permission. 
+Staff user can't appDeactivate app which has more permissions than he has for his own.
```
Comment (maarcingebala) [inline docs/developer/extending/apps.mdx:519]:
```suggestion
Staff user can't deactivate an app which has more permissions than he has for his own. 
```

File: docs/developer/extending/apps.mdx [hunk 3]
```diff
@@ -102,26 +424,142 @@ mutation {
...
+  }
+}
+```
+### Deactivating an App
+
+To deactivate an app, use the `appDeactivate` mutation. This mutation is only available to staff users with the `MANAGE_APPS` permission. 
+Staff user can't appDeactivate app which has more permissions than he has for his own. 
+
+:::note
+The deactivating app will disable all auth-tokens generated for an app.
```
Comment (maarcingebala) [inline docs/developer/extending/apps.mdx:522]:
```suggestion
Deactivating an app will disable all auth-tokens generated for an app. 
```

File: docs/developer/extending/apps.mdx [hunk 4]
```diff
@@ -102,26 +424,142 @@ mutation {
...
+    appErrors{
+      field
+      message
+      code
+    }
+  }
+}
+```
+
+The query returns simillar response:
```
Comment (maarcingebala) [inline docs/developer/extending/apps.mdx:502]:
```suggestion
The query returns a similar response:
```

File: docs/developer/extending/apps.mdx [hunk 5]
```diff
@@ -102,26 +424,142 @@ mutation {
...
+        }
+        
+      }
+    }
+  }
+}
+```
+
+
+The query returns simillar response:
```
Comment (maarcingebala) [inline docs/developer/extending/apps.mdx:453]:
```suggestion
The query returns a similar response:
```

File: docs/developer/extending/apps.mdx [hunk 6]
```diff
@@ -1,8 +1,343 @@
...
+```graphql
+{
+  appsInstallations{
+    id
+    status
+    appName
+  }
+}
+```
+The query returns simillar response:
```
Comment (maarcingebala) [inline docs/developer/extending/apps.mdx:280]:
```suggestion
The query returns a similar response:
```

File: docs/developer/extending/apps.mdx (hunk 2 above)
Comment (maarcingebala) [reply docs/developer/extending/apps.mdx:519]: Also, to not assume gender ("he") you can say "Staff users [...] they have for their own" :)

File: docs/developer/extending/apps.mdx [hunk 7]
```diff
@@ -73,9 +397,7 @@ Creating an app with `appCreate` mutation creates the default authorization toke
 
 ```graphql {2-4}
```
Comment (patrys) [inline docs/developer/extending/apps.mdx:398]: You will need to update the highlighted range as well.

File: docs/developer/extending/apps.mdx (hunk 2 above)
Comment (patrys) [reply docs/developer/extending/apps.mdx:519]: Yes, always use the neutral "they"/"their" instead of "he"/"him"/"his".

**Pull request merged**